okhttp-android-support/src/test/java/okhttp3/internal/huc/CacheAdapterTest.java

```diff
@@ -116,7 +116,7 @@
     };
     setInternalCache(new CacheAdapter(responseCache));
     client = client.newBuilder()
-        .sslSocketFactory(sslContext.getSocketFactory())
+        .sslSocketFactory(sslContextBuilder.socketFactory(), sslContextBuilder.trustManager())
```

Comment:
... if we make `SslClient` or whatever, these can be public final member fields instead of methods, which feels neater